src/ntfsck.c

```diff
@@ -2381,6 +2381,8 @@ static int ntfsck_scan_index_entries_btree(ntfs_volume *vol)
 				goto err_continue;
 			}
 			prev = next;
+		} else if (ret < 0) {
+			goto err_continue;
 		}
```

Comment:
I think it's better to describe about return value in ntfs_index_netry_inconsistent()
when return positive value, or when return negative value. what's difference between both. etc.